common/dispatcher.hpp

```diff
@@ -54,6 +54,17 @@ bool execute_action(uint64_t action, void (Q::*func)(Args...)) {
 #define APP_DOMAIN_API(TYPENAME, MEMBERS) \
     BOOST_PP_SEQ_FOR_EACH(APP_DOMAIN_API_CALL, TYPENAME, MEMBERS)
 
+// Temporary workaround
+#ifdef HANDLE_TRANSFER
+#   define HANDLE_TRANSFER_ACTION(T) if (action == N(transfer) && code == golos::config::token_name) { \
+    eosio_assert(1, "jo");\
```

Comment:
Is this assert needed?

Reply:
nop, it's from debug.
thole this temporary block will be removed in #283